argus/backend/service/notification_manager.py

```diff
@@ -11,19 +13,53 @@ class NotificationManagerException(Exception):
 
 
 class NotificationManagerService:
+
+    def __init__(self) -> None:
+        self.notification_services = []
+        self.attach_notification_sender(ArgusDBNotificationSaver())
+        self.attach_notification_sender(EmailNotificationServiceSender())
+
+    def attach_notification_sender(self, service):
```

Comment:
this method is redundant, in `__init__` we could populate `self.notification_services` without it.
It is better to add minimum required public methods.